Training_node/first_project/app/controllers/users.js

```diff
@@ -81,18 +77,15 @@ exports.create = (req, res, next) => {
         .then(newPassword => {
           user.password = newPassword;
           return User.getByEmail(user.email).then(exist => {
-            if (!exist) {
-              return User.createModel(user)
-                .then(u => {
-                  res.send(user.email);
-                  res.status(200);
-                  res.end();
-                })
-                .catch(err => {
-                  next(err);
-                });
+            if (admin) {
+              user.admin = true;
+              return exist ? setAdmin(exist, res, next) : createModel(user, res, next);
             } else {
-              next(errors.existsUser);
+              if (!exist) {
```

Comment:
you dont need this

Reply:
Why ? If the user exist and you do not want to create an admin, it is an error